src/config.rs

```diff
@@ -1759,28 +1766,6 @@ mod readpool_tests {
 
     #[test]
     fn test_unified_enabled() {
-        // Allow invalid storage and coprocessor config when yatp is used.
```

Comment:
It's deleted because we are now specifying whether to use the unified read pool in storage and coprocessor separately